src/models/notifications.js

```diff
@@ -1,59 +0,0 @@
-'use strict'
-
-const log = require('../services/log')('subscriptions')
-const uri = require('../services/uriManager')
-const NotFoundError = require('five-bells-shared/errors/not-found-error')
-const UnauthorizedError = require('five-bells-shared/errors/unauthorized-error')
-const Notification = require('./db/notification').Notification
-const Transfer = require('./db/transfer').Transfer
-const Fulfillment = require('./db/conditionFulfillment').ConditionFulfillment
-const subscriptions = require('./db/subscriptions')
-const subscriptionUtils = require('../lib/subscriptionUtils')
-const transferDictionary = require('five-bells-shared').TransferStateDictionary
-
-const transferStates = transferDictionary.transferStates
-
-function * getNotification (subscriptionId, notificationId, requestingUser) {
```

Comment:
If we're removing the notion of notification ids and you would just need to request a transfer state that would probably be sufficient